content/applications/inventory_and_mrp/manufacturing/basic_setup/bill_configuration.rst

```diff
@@ -2,98 +2,259 @@
+clicking :guilabel:`Add a line`. From the :guilabel:`Components` drop-down menu, select from
+existing products or create a new product by typing the name and selecting either the
+:guilabel:`Create " "` option to quickly add the line item, or the :guilabel:`Create and edit...`
+option to add the component and continue to its configuration form.
+
+.. image:: bill_configuration/component.png
    :align: center
-   :alt: Set up a Bill of Materials.
+   :alt: Add a component by selecting it from the drop-down menu.
 
-Specify a bill of materials (BoM) for a product variant
--------------------------------------------------------
+Optionally, access additional fields by clicking the :icon:`oi-settings-adjust` :guilabel:`(settings
+adjust)` icon to the far-right of the :guilabel:`Components` tab. Tick the checkboxes for the
+following features to enable these columns:
 
-:abbr:`BoMs (Bills of Materials)` can also be assigned to specific *Product Variants*, with two
-setup options available to choose from.
+- :guilabel:`Apply on Variants`: specify which :doc:`product variant
+  <../advanced_configuration/product_variants>` each component is used in. When the field is left
+  blank, the component is used in all product variants.
 
-.. note::
-   In order to assign :abbr:`BoMs (Bills of Materials)` to product variants, the product's variant
-   attributes must already be configured on the product form.
+.. _manufacturing/basic_setup/consumed-in-operation:
+
+- :guilabel:`Consumed in Operation`: specify the operation using the component. Useful for
+  determining :ref:`manufacturing readiness <manufacturing/basic_setup/manufacturing-readiness>`.
+- :guilabel:`Manual Consumption`: tick the checkbox to force operators to check the
+  :guilabel:`Consumed` checkbox on a manufacturing order (MO).
+
+  .. image:: bill_configuration/consumed-field.png
+     :align: center
+     :alt: Show a manufacturing order, highlighting the *Consumed* field.
+
+  Not doing so triggers the :guilabel:`Consumption Warning` error message, where the consumed
+  component quantity must be manually inputted. Otherwise, the operation cannot be completed.
+
+  .. image:: bill_configuration/consumption-warning.png
+     :align: center
+     :alt: Show the consumption warning error message.
+
+.. _manufacturing/basic_setup/setup-operations:
+
+Operations
+----------
 
-The first method is to create one :abbr:`BoM (Bill of Materials)` per variant by creating a new
-:abbr:`BoM (Bill of Materials)` and specifying the :guilabel:`Product Variant`. The second method
-is to create one master :abbr:`BoM (Bill of Materials)` that contains all of the components, and
-specify which variant each component applies to using the :guilabel:`Apply on Variants` column.
+Add an :guilabel:`Operation` to a |BOM| to specify instructions for production and register time
+spent on an operation. To use this feature, first enable the *Work Orders* feature by going to
```

Comment:
Nitpick– the text "Operation" is not visible on the current UI and because in this context you are referring to the Operations feature, and not necessarily the Operations tab
```suggestion
Add an *Operation* to a |BOM| to specify instructions for production and register time spent on an
operation. To use this feature, first enable the *Work Orders* feature by going to
```